tests/library/inspector/cli-codegen-1.spec.ts

```diff
@@ -19,6 +19,7 @@ import type { ConsoleMessage } from 'playwright';
 
 test.describe('cli codegen', () => {
   test.skip(({ mode }) => mode !== 'default');
+  test.skip(({ trace }) => trace === 'on' && test.info().project.name.endsWith('-codegen-mode-trace'));
```

Comment:
Here and everywhere:

```suggestion
  test.skip(({ trace, codegenMode }) => trace === 'on' && codegenMode === 'trace-events');
```